definitions/output/reports/cwv_tech_technologies.js

```diff
@@ -6,19 +6,68 @@ publish('cwv_tech_technologies', {
   tags: ['crux_ready']
 }).query(ctx => `
 /* {"dataform_trigger": "report_cwv_tech_complete", "name": "technologies", "type": "dict"} */
+WITH pages AS (
+  SELECT DISTINCT
+    client,
+    root_page AS origin,
+    tech.technology
+  FROM ${ctx.ref('crawl', 'pages')},
+    UNNEST(technologies) AS tech
+  WHERE
+    date = '${pastMonth}'
+    ${constants.devRankFilter}
+), tech_origins AS (
+  SELECT
+    client,
+    technology,
+    COUNT(origin) AS origins
+  FROM pages
+  GROUP BY
+    client,
+    technology
+), technologies AS (
+  SELECT
+    name AS technology,
+    description,
+    ARRAY_TO_STRING(categories, ', ') AS category,
```

Comment:
Will this always be consistently ordered?

Reply:
It preserves order from wappalyzer configs, if that's what you mean.
It's not always ASC.

Reply:
Will it be OK?

Reply:
Maybe, but probably can't hurt to explicitly sort here right?